packages/storage-plus/src/lib.rs

```diff
@@ -42,9 +42,34 @@ pub use prefix::{range_with_prefix, Prefix};
 #[cfg(feature = "iterator")]
 pub use snapshot::{SnapshotItem, SnapshotMap, Strategy};
 
+// cw_storage_macro reexports
 #[cfg(all(feature = "iterator", feature = "macro"))]
 #[macro_use]
 extern crate cw_storage_macro;
 #[cfg(all(feature = "iterator", feature = "macro"))]
-#[doc(hidden)]
+/// Auto generate an `IndexList` impl for your indexes struct.
+///
+/// # Example
+///
+/// ```rust
+/// use cosmwasm_std::Addr;
+/// use cw_storage_plus::{MultiIndex, UniqueIndex, index_list};
+/// use serde::{Serialize, Deserialize};
+///
+/// #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+/// struct TestStruct {
+///     id: u64,
+///     id2: u32,
+///     addr: Addr,
+/// }
+///
+/// #[index_list(TestStruct)] // <- Add this line right here.
+/// struct TestIndexes<'a> {
+///     id: MultiIndex<'a, u32, TestStruct, u64>,
+///     addr: UniqueIndex<'a, Addr, TestStruct>,
+/// }
+/// ```
+///
+pub use cw_storage_macro::index_list;
+#[cfg(all(feature = "iterator", feature = "macro"))]
 pub use cw_storage_macro::*;
```

Comment:
Ah, I think we can get rid of this!

Reply:
I added it in case we forget to reexport new items that might get added to the macro crate in the future, but I guess we'd have to bump at least the version anyway in that case.

Reply:
Yeah. This could also cause us to reexport something we didn't want to by accident. And we want to document everything we reexport from the macro crate anyway.